pandas/_libs/groupby.pyx

```diff
@@ -707,6 +707,7 @@ def group_sum(
     uint8_t[:, ::1] result_mask=None,
     Py_ssize_t min_count=0,
     bint is_datetimelike=False,
+    object initial=0,
```

Comment:
Just confirming, will this only ever be `0` or `""`?

Reply:
At the moment, yes (since this is internal, we have control over what we pass to it). I could in theory make the typing more specific, I assume, with Literal in python. But here in cython that is not possible?